x-pack/plugins/security_solution/cypress/e2e/detection_alerts/investigate_in_timeline.cy.ts

```diff
@@ -5,36 +5,91 @@
  * 2.0.
  */
 
+import { closeTimeline } from '../../tasks/timeline';
 import { getNewRule } from '../../objects/rule';
-import { PROVIDER_BADGE } from '../../screens/timeline';
+import { PROVIDER_BADGE, QUERY_TAB_BUTTON, TIMELINE_TITLE } from '../../screens/timeline';
 
-import { investigateFirstAlertInTimeline } from '../../tasks/alerts';
+import { expandFirstAlert, investigateFirstAlertInTimeline } from '../../tasks/alerts';
 import { createCustomRuleEnabled } from '../../tasks/api_calls/rules';
 import { cleanKibana } from '../../tasks/common';
 import { waitForAlertsToPopulate } from '../../tasks/create_new_rule';
 import { login, visit } from '../../tasks/login';
 
 import { ALERTS_URL } from '../../urls/navigation';
+import {
+  ALERT_FLYOUT,
+  INSIGHTS_INVESTIGATE_ANCESTRY_ALERTS_IN_TIMELINE_BUTTON,
+  INSIGHTS_INVESTIGATE_IN_TIMELINE_BUTTON,
+  INSIGHTS_RELATED_ALERTS_BY_ANCESTRY,
+  INSIGHTS_RELATED_ALERTS_BY_SESSION,
+  SUMMARY_VIEW_INVESTIGATE_IN_TIMELINE_BUTTON,
+  SUMMARY_VIEW_PREVALENCE_CELL,
+} from '../../screens/alerts_details';
+import { verifyInsightCount } from '../../tasks/alerts_details';
 
-describe('Alerts timeline', () => {
+describe('Investigate in timeline', { testIsolation: false }, () => {
   before(() => {
     cleanKibana();
     login();
     createCustomRuleEnabled(getNewRule());
-  });
-  beforeEach(() => {
     visit(ALERTS_URL);
     waitForAlertsToPopulate();
   });
 
-  it('Investigate alert in default timeline', () => {
-    investigateFirstAlertInTimeline();
-    cy.get(PROVIDER_BADGE)
-      .first()
-      .invoke('text')
-      .then((eventId) => {
-        investigateFirstAlertInTimeline();
-        cy.get(PROVIDER_BADGE).filter(':visible').should('have.text', eventId);
+  describe('From alerts table', () => {
+    after(() => {
+      closeTimeline();
+    });
+
+    it('should open new timeline from alerts table', () => {
+      investigateFirstAlertInTimeline();
+      cy.get(PROVIDER_BADGE)
+        .first()
+        .invoke('text')
+        .then((eventId) => {
+          investigateFirstAlertInTimeline();
+          cy.get(PROVIDER_BADGE).filter(':visible').should('have.text', eventId);
+        });
+    });
+  });
+
+  describe('From alerts details flyout', () => {
+    before(() => {
+      expandFirstAlert();
+    });
+
+    afterEach(() => {
+      closeTimeline();
```

Comment:
@PhilippeOberti .. should include `closeFlyout` as well?

Reply:
I thought about it but I feel like the tests are verifying the content of the flyout, not the fact that it can be opened. Not closing it between tests make the tests run a tiny bit faster... which is why I decided against closing it